### PR TITLE
Update logs loading description

### LINKED ISSUE #4416: Remove the little "100% pure logs" or whatever text when searching for logs.
this is strange when searching back in time. Can we replace it with just a small spinner?

### PR DESCRIPTION
**Pull request by OfTheDelmer**
Closes #4416 

_Briefly describe your proposed changes:_
Updates the log table loading message to display the `lower` and `upper` time bounds being searched

_What was the problem?_
Logs table was displaying time ago from now when it was searching which could be ambiguous or make the search seem to stall. The logs loading description also displayed random messages that weren't truly descriptive of the search.

_What was the solution?_
Remove the random log messages. 
Display the actual upper and lower times being used for search. 
Clean up some logs table time formatting changes by adding the `formatTime` util.

  - [x] CHANGELOG.md updated with a link to the PR (not the Issue)
  - [x] Rebased/mergeable
  - [x] Tests pass

log_searching
logs-search-anim

### CODE REVIEW

File: ui/src/style/pages/logs-viewer.scss [hunk 1]
```diff
@@ -334,4 +334,90 @@ $logs-viewer-gutter: 60px;
...
+  animation: magRotateA 1.8s infinite linear;
+}
+.logs-viewer--graphic-log {
+  background-image: url('../../assets/images/log.svg');
+  width: 50px;
+  height: 50px;
+  transform: translate(-50%,-50%);
+}
+.logs-viewer--graphic-magnifier-b {
+  background-image: url('../../assets/images/magnifier.svg');
```
Comment (AlirieGray) [inline ui/src/style/pages/logs-viewer.scss]: should we potentially use absolute paths?

**Comment by lukevmorris** [comment]
Has css so adding @fntlnz for extra review

File: ui/src/style/pages/logs-viewer.scss (hunk 1 above)
Comment (OfTheDelmer) [reply ui/src/style/pages/logs-viewer.scss]: It looks like this a pattern in the styles for various components

File: ui/src/logs/components/loading_status/LoadingStatus.tsx [hunk 2]
```diff
@@ -30,32 +26,64 @@ class LoadingStatus extends PureComponent<Props> {
...
       case SearchStatus.UpdatingSource:
-        return 'Searching updated source'
+        return 'Searching updated source...'
       case SearchStatus.UpdatingNamespace:
-        return 'Searching updated namespace'
+        return 'Searching updated namespace...'
       case SearchStatus.Loading:
+        return 'Searching...'
       default:
         return 'Searching'
```
Comment (OfTheDelmer) [inline ui/src/logs/components/loading_status/LoadingStatus.tsx]: @alexpaxton do we want the default to also be `Searching...`

**Pull request merged**